Common Programming Concepts/Variables/Constant Type/task.md

```diff
@@ -8,9 +8,10 @@ variable available; constants.
 </div>
 
 <div class="hint">
+
 Constants are always immutable and they are declared with keyword <code>const</code> rather
```

Comment:
Constants are always immutable and are declared using the <code>const</code> keyword instead of <code>let</code>.